Automattic/jetpack
# AI Assistant: Tweak Gutenberg prompt and add block processing structure

Fixes #

## Proposed changes:

* Changes the prompt with Gutenberg-specific rules
* Adds block processing structure to try to fix invalid blocks

### Other information:


## Jetpack product discussion


pe4Cmx-1vJ-p2

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Use the AI Assistant with Gutenberg syntax and version 4 options turned on
* Check that tables and covers are inserted
* Check the debug lines
* If those blocks were invalid, some debug lines should be displayed with the steps taken to fix the block
